libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -991,6 +1008,14 @@ void ff_dnn_free_model_tf(DNNModel **model)
 
     if (*model){
         tf_model = (*model)->model;
+        while (ff_safe_queue_size(tf_model->request_queue) != 0) {
+            TFRequestItem *item = ff_safe_queue_pop_front(tf_model->request_queue);
+            tf_free_request(item->infer_request);
+            av_freep(&item->infer_request);
+            av_freep(&item);
+        }
+        ff_safe_queue_destroy(tf_model->request_queue);
```

Comment:
I did a quick review of patch: lavfi/dnn_backend_tf: Request-based Execution. I did not find obvious issues, the basic code logic is right. So now no comment for this patch.

Reply:
Okay, thanks.

Reply:
btw, I'm thinking the name of inference queue / InferenceItem.  Looks that they are not the best because of the existing of batch mode.  One inference item is not for one inference API call to the framework (TF/OpenVINO/Native), that is what requestItem/infer_request/request_queue does.  How about to change InferenceItem to SubtaskItem?  Do you have a better naming?  @Semmer2  @shubhanshu02  

The rename can be done after all of these patches in this patch set are accepted in upstream.

Reply:
Yes, the name `InferenceItem` is a bit misleading. The name `SubTaskItem`, as you suggested, seems good for naming since each `InferenceItem` is a part of a `TaskItem`.

Reply:
> Yes, the name `InferenceItem` is a bit misleading. The name `SubTaskItem`, as you suggested, seems good for naming since each `InferenceItem` is a part of a `TaskItem`.

@Semmer2 any comment? if no, we can change to SubtaskItem, after shubhanshu02's patches, which reply on InferenceItem, are merged.

Reply:
or LastLevelTaskItem?